samples/CallWithChat/src/app/utils/utils.ts

```diff
@@ -1,50 +1,6 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-declare let __BUILDTIME__: string; // Injected by webpack
```

Comment:
This was all duplicated code that wasn't being used. Webpack injections were defined and used from AppUtils.ts